engine/src/main/java/org/terasology/world/chunks/localChunkProvider/ChunkCache.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.world.chunks.localChunkProvider;
+
+import org.terasology.math.geom.Vector3i;
+import org.terasology.world.chunks.Chunk;
+
+import java.util.Collection;
+import java.util.Iterator;
+
+interface ChunkCache {
```

Comment:
I think interfaces and classes should have a small javadoc that informs about purpose.